tools/isoplot/isoplot.xml

```diff
@@ -0,0 +1,155 @@
+        <param name='template_file' type='data' format='csv, xlsx' multiple='true' label='Template file' argument='--template'/>
+        <!--param name='run_name' type='text' label='Run name' /-->
+        <param name='format' type='select' label='Plot format'>
+            <option value='pdf'>pdf</option>
+            <option value='jpeg'>jpeg</option>
+            <option value='png'>png</option>
+            <option value='svg'>svg</option>
+            <option value='html'>html</option>
+        </param>
+        <param name='value' type='select' label='Data to plot' >
+            <option value='corrected_area'>Corrected Area</option>
+            <option value='isotopologue_fraction'>Isotopologue Fraction</option>
+            <option value='mean_enrichment'>Mean Enrichment</option>
+        </param>
+        <section name='SP' title='Select static plots to generate'>
+            <param type='boolean' name='generate_barplot' label='Flag to generate barplots' argument='-bp'
+                   optional='true'/>
+            <param type='boolean' name='generate_meanplot' label='Flag to generate meaned barplots' argument='-mb'
+                   optional='true'/>
+            <!--<param type='boolean' name='generate_areaplot' label='Flag to generate stacked area plots'
+                   argument='-sa' optional='true'/>/-->
+            <param type='boolean' name='generate_heatmap' label='Flag to generate heatmap' argument='-hm'
+                   optional='true'/>
+            <param type='boolean' name='generate_clustermap' label='Flag to generate clustermap' argument='-cm'
+                   optional='true'/>
+        </section>
+        <section name='IP' title='Select interactive plots to generate'>
+            <param type='boolean' name='generate_barplot' label='Flag to generate barplots' argument='-IB'
+                   optional='true'/>
+            <param type='boolean' name='generate_meanplot' label='Flag to generate meaned barplots' argument='-IM'
+                   optional='true'/>
+           <!-- <param type='boolean' name='generate_areaplot' label='Flag to generate stacked area plots' argument='-IS'
+                   optional='true'/>-->
+            <param type='boolean' name='generate_heatmap' label='Flag to generate heatmap' argument='-HM'
+                   optional='true'/>
+        </section>
+        <param type='boolean' name='stack' label='Flag to unstack bars in barplots' argument='-s' optional='true'/>
+        <param type='boolean' name='verbosity' label='Flag to get debug information' argument='-v' optional='true'/>
+        <param type='boolean' name='annotation' label='Flag to add annotations on the maps' argument='-a'
+               optional='true'/>
+    </inputs>
+    
+    <outputs>
+        <data name='output1' format='csv' label='Formatted data used for plotting' />
+        <data name='output2' format='zip' label='Resulting plots' />

```

Comment:
zip is not very useful in Galaxy, would a collection of images work for you as well?

Reply:
Thank you for the review! I have made most of the changes you notified me about, except for this one. I have trouble understanding how collections work and so creating a zip archive was my workaround. Is there any particular reason why a collection should be used over a zip file?

Regards

Reply:
Collections can be reused by other tools, zip not. galaxy can also parallelize collections later one and more importantly users can look at collections, visualise the content etc. You can not look into zip without downloading it.

Reply:
Thank you for the info, I will look into it and work on it for a later release of the wrapper!

Reply:
Why not now :)